azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/ArtifactHandlerImplV2.java

```diff
@@ -31,7 +37,14 @@
     private static final int MAX_RETRY_TIMES = 3;
     private static final String ALWAYS_DEPLOY_PROPERTY = "alwaysDeploy";
 
-    public static final String RENAMING_MESSAGE = "Renaming %s to %s";
+    private static final String WEB_CONFIG = "web.config";
+    private static final String RENAMING_MESSAGE = "Renaming %s to %s";
+    private static final String RENAMING_FAILED_MESSAGE = "Failed to rename artifact to %s, which is required in Java SE environment, " +
+            "please refer https://docs.microsoft.com/en-us/azure/app-service/containers/configure-language-java#set-java-runtime-options for more information.";
+    public static final String NO_EXECUTABLE_JAR = "No executable jar found in <resources>, please check the configuration";
+    public static final String MULTI_FINAL_NAME_JARS = "Multi executable jars with final name %s found in <resources>, please check the configuration";
```

Comment:
How can this happen? there should be only one final jar built during package.

Reply:
I want to scan artifacts in sub folders of targer folder but app service will not detect them, will fixed it.